I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 89, in script
    python:
  File "game/script.rpy", line 97, in <module>
    renpy.call_screen("Select_Quest_Screen",each)
  File "renpy/common/00action_other.rpy", line 537, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/gameplay/global_variables.rpy", line 63, in assign_quest_to_player
    print(player.my_quest.name)
AttributeError: 'Player' object has no attribute 'name'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 89, in script
    python:
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\ast.py", line 914, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\python.py", line 2028, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 97, in <module>
    renpy.call_screen("Select_Quest_Screen",each)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\exports.py", line 2905, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\ui.py", line 297, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\core.py", line 2702, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\core.py", line 3518, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\layout.py", line 998, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\layout.py", line 998, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\layout.py", line 998, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\screen.py", line 714, in event
    rv = self.child.event(ev, x, y, st)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\layout.py", line 998, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\layout.py", line 998, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\behavior.py", line 962, in event
    return handle_click(self.clicked)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\behavior.py", line 897, in handle_click
    rv = run(action)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\behavior.py", line 313, in run
    new_rv = run(i, *args, **kwargs)
  File "C:\Users\Michael\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\behavior.py", line 320, in run
    return action(*args, **kwargs)
  File "renpy/common/00action_other.rpy", line 537, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/gameplay/global_variables.rpy", line 63, in assign_quest_to_player
    print(player.my_quest.name)
AttributeError: 'Player' object has no attribute 'name'

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Tricks and Treats 1.0
Mon Oct 26 17:05:20 2020
